src/Monolog/Handler/TelegramBotHandler.php

```diff
@@ -46,6 +55,26 @@ class TelegramBotHandler extends AbstractProcessingHandler
      */
     private $channel;
 
+    /**
+     * The kind of formatting that is used for the message.
+     * See available options at https://core.telegram.org/bots/api#formatting-options
+     * or in AVAILABLE_PARSE_MODES
+     * @var string
+     */
+    private $parseMode;
+
+    /**
+     * Disables link previews for links in the message.
+     * @var bool
+     */
+    private $disableWebPagePreview;
+
+    /**
+     * Sends the message silently. Users will receive a notification with no sound.
+     * @var bool
```

Comment:
As far as I can tell all 3 vars will default to null and if not configured will remain so. Those `@var` should probably include `|null`.